Classes/Domain/Model/Order/ProductAdditional.php

```diff
@@ -14,27 +14,9 @@
 
 class ProductAdditional extends AbstractEntity
 {
-    protected string $additionalType;
-
-    protected string $additionalKey;
-
-    protected string $additionalValue;
-
-    protected string $additionalData = '';
-
     protected string $additional;
 
-    public function __construct(
-        string $type,
-        string $key,
-        string $value,
-        string $data = ''
-    ) {
-        $this->additionalType = $type;
-        $this->additionalKey = $key;
-        $this->additionalValue = $value;
-        $this->additionalData = $data;
-    }
+    public function __construct(protected string $additionalType, protected string $additionalKey, protected string $additionalValue, protected string $additionalData = '') {}
```

Comment:
Might be better to split it on multiple lines?

Reply:
Fix this with e91a033.